### PR TITLE
Week 4 Quiz submission from asharda-stat6250

### PR DESCRIPTION
**Pull request by asharda-stat6863**
Week 4 Quiz submission from asharda-stat6250

### CODE REVIEW

File: week-04/README.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ The instructor will then review the pull request and make comments should furthe
 - *Question* (akrishnamurthy-stat6250): How should format name be defined for character and numeric data ?
 - Question (lceballos-stat6250): Why can't the format with the VALUE statement end in a period?
 - Question (aguenane-stat6250): What is the general form of the VALUE statement?
+- *Answer* (asharda-stat6250):
```
Comment (prof-lankham) [inline week-04/README.md]: Please update code sample with code fence; see https://help.github.com/articles/creating-and-highlighting-code-blocks/

File: week-04/README.md [hunk 2]
```diff
@@ -79,6 +85,7 @@ The instructor will then review the pull request and make comments should furthe
 - *Question* (ldai4-stat6250): How do we use VALUE statement to display one or more values?
 - *Answer* (ldai4-stat6250): The statement begins with the keyword VALUE and ends with a semicolon after all the labels have been defined.
 - Question (nshrivastava2-stat6250): How can we use the format procedure to create the formats? 
+ - *Answer* (asharda-stat6250):PROC FORMAT
```
Comment (prof-lankham) [inline week-04/README.md]: This answer should be expanded

File: week-04/README.md [hunk 3]
```diff
@@ -245,6 +256,8 @@ proc means data=mydata &mmse.;
 run;
 ```
 - Question (who7-stat6250): What is the main difference between PROC MEAN and PROC SUMMARY?
+- *Answer* (asharda-stat6250): The difference between the two procedures is that PROC MEANS produces a report by default 
+- 282 - (remember that you can use the NOPRINT option to suppress the default report). By contrast, to produce a report in PROC SUMMARY, you must include a PRINT option in the PROC SUMMARY statement.
```
Comment (prof-lankham) [inline week-04/README.md]: Please remove line break from this answer

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Please update PR and DM me in Slack with PR URL for re-evaluation

File: week-04/README.md [hunk 4]
```diff
@@ -79,6 +80,8 @@ The instructor will then review the pull request and make comments should furthe
 - *Question* (ldai4-stat6250): How do we use VALUE statement to display one or more values?
 - *Answer* (ldai4-stat6250): The statement begins with the keyword VALUE and ends with a semicolon after all the labels have been defined.
 - Question (nshrivastava2-stat6250): How can we use the format procedure to create the formats? 
+ - *Answer* (asharda-stat6250):PROC FORMAT.The PROC FORMAT block contains three statements each terminating 
+wit a semicolon. The first statement calls PROC FORMAT. The second statement is a value clause that defines the name and contents of the user-defined format. The final statement is the run statement which, though not technically required, is good programming practice nonetheless.
```
Comment (prof-lankham) [inline week-04/README.md]: Remove line break here.

File: week-04/README.md [hunk 5]
```diff
@@ -245,6 +252,8 @@ proc means data=mydata &mmse.;
 run;
 ```
 - Question (who7-stat6250): What is the main difference between PROC MEAN and PROC SUMMARY?
+- *Answer* (asharda-stat6250): The difference between the two procedures is that PROC MEANS produces a report by default 
+- 282 - (remember that you can use the NOPRINT option to suppress the default report). By contrast, to produce a report in PROC SUMMARY, you must include a PRINT option in the PROC SUMMARY statement.
```
Comment (prof-lankham) [inline week-04/README.md]: Remove line break here.

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Before this PR can be merged, please remove line breaks from answers and DM me in Slack with PR URL.

**Comment by asharda-stat6863** [comment]
Deleted branch as was unable to fork,closing it.submitted a new PR

**Pull request closed without merging**